docs/tutorials/discovery-fabric-network.md

```diff
@@ -0,0 +1,245 @@
+
+This tutorial illustrates the use of the service discovery by the Hyperledger Fabric Node.js fabric-network SDK.
+
+For more information on:
+* [Fabric Service Discovery](https://hyperledger-fabric.readthedocs.io/en/release-1.4/discovery-overview.html#how-service-discovery-works-in-fabric)
+
+The following assumes an understanding of the Hyperledger Fabric network
+(orderers and peers) and the Fabric discovery service.
+and of Node application development, including the use of the
+Javascript `promise` and `async await`.
+
+### Overview
+The service discovery provided by the Hyperledger Fabric helps an application
+understand the current view of the network. Service discovery also has insight
+into the endorsement policies of chaincodes and is able to provide lists of
+peers that are currently active on the network that could be used to endorse a
+proposal.
+To use the discovery service, the application will have to define just one peer.
```

Comment:
...will have to **define** just one peer.
Hmm, since fabric-network doesn't define peers the way fabric-client did, I wonder if we should say something like ...will have to be configured with just one peer (in the CCP)